include/maths/CKdTree.h

```diff
@@ -442,6 +484,13 @@ class CKdTree {
         f(node);
     }
 
+    //! Get the next coordinate.
+    inline std::size_t nextCoordinate(std::size_t coordinate) const {
```

Comment:
I'm not convinced it's worth specifying `inline` here.  The method is small so the compiler will almost certainly inline it anyway since it has the definition.

Reply:
Equally I don't think it hurts and expresses intention clearly that this shouldn't incur a function call.

Reply:
My inclination is to leave this as annotated inline unless you feel strongly.